arbeitszeit/use_cases/remove_worker_from_company.py

```diff
@@ -46,20 +46,31 @@ def remove_worker_from_company(self, request: Request) -> Response:
         if not workplace:
             return Response(Response.RejectionReason.not_workplace_of_worker)
         workplace_record.remove_worker(worker.id)
-        self._notify_worker(worker, workplace)
+        self._notify_worker_and_company(worker, workplace)
         return Response(None)
 
-    def _notify_worker(self, worker: Member, workplace: Company) -> None:
-        record = (
+    def _notify_worker_and_company(self, worker: Member, workplace: Company) -> None:
+        worker_record = (
```

Comment:
You might consider joining member and workplace in line 41 and 44 already with the respective email addresses, which would save us two calls to the database and the check if worker and company exist.